feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/followrequest/FollowerRequestFragment.java

```diff
@@ -57,6 +59,21 @@ public void onChanged(@Nullable String s) {
 
         return root;
     }
+
+    private void storeFollowerPermission(String friendName){
+        //TODO: pass filename variable from MainActivity
+        SharedPreferences mySharedPreferences = this.getActivity().getSharedPreferences("Xiaole", Context.MODE_PRIVATE);
```

Comment:
Is it possible to set the sharedPreference key to "USERNAME" or something of the sort? That way it will work regardless of what the actual username is